addons/pharma/stringtable.xml

```diff
@@ -2536,7 +2537,7 @@
         </Key>
         <Key ID="STR_kat_pharma_Pervitin_mid2">
             <English>You gain a sudden rush!</English>
-            <German>Du bekommst einen spontanan Rush...</German>
+            <German>Du bekommst einen spontanen Schub...</German>
```

Comment:
"Schub" is to undescribing and will confuse the user. Rush is at this point a well know word in german as well.

Reply:
Not sure if I would agree, since "Schub" and "Rush" literally means the same, whereas "Rush" is the anglicized word. But the Word Rush is more known and used in the German Drug scene, so it might fit better...
I undid my Change anyhow.